postgres/src/main/java/org/jdbi/v3/postgres/BlobInputStreamArgumentFactory.java

```diff
@@ -0,0 +1,38 @@
+package org.jdbi.v3.postgres;
+
+import java.io.InputStream;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import org.jdbi.v3.core.argument.AbstractArgumentFactory;
+import org.jdbi.v3.core.argument.Argument;
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.jdbi.v3.core.statement.StatementContext;
+
+class BlobInputStreamArgumentFactory extends AbstractArgumentFactory<InputStream> {
```

Comment:
Are there no other potential/reasonable use cases for an InputStream in queries, like are these postgres lobs equivalent to general lobs for all dbs? If there are uses that don't mix with this, this may need a qualifier

Reply:
It only does that for now.  We can always change the name later if the class ends up doing more, it's not API.

Reply:
I meant if InputStreams couldn't serve purposes not implemented by this class, like different kinds of lobs or something.

Reply:
If / when that happens we might have to differentiate by a qualifier annotation?